arangod/Cluster/ClusterInfo.cpp

```diff
@@ -4436,7 +4414,7 @@ void ClusterInfo::invalidateCurrentMappings() {
 //////////////////////////////////////////////////////////////////////////////
 
 void ClusterInfo::invalidateCurrent() {
-  {
+/*  {
```

Comment:
What about `invalidateCurrentCoordinators` and `invalidateCurrentMappings`? Are they also updated by `loadCurrent`? Should we maybe leave their invalidation?